src/amuse/couple/multiples.py

```diff
@@ -1781,7 +1781,7 @@ def find_nn3(plist, field, G):
         phimin = 0.0 | nbody_system.energy
         dxmin = 1.e30
         pmin = None
-        j = numpy.argmin(phi)
+        j = numpy.argmin(phi.number)
```

Comment:
the fix is ok; did you have it failing? it works here...
( a safe alternative is j=phi.argmin())